ethpm/utils/backend.py

```diff
@@ -0,0 +1,73 @@
+from typing import Generator, Type
+
+from eth_utils import to_tuple
+
+from ethpm.backends.base import BaseURIBackend
+from ethpm.backends.ipfs import (
+    DummyIPFSBackend,
+    InfuraIPFSBackend,
+    IPFSGatewayBackend,
+    LocalIPFSBackend,
+    get_ipfs_backend_class,
+)
+from ethpm.backends.registry import RegistryURIBackend
+from ethpm.exceptions import CannotHandleURI
+
+URI_BACKENDS = [
+    IPFSGatewayBackend,
+    InfuraIPFSBackend,
+    DummyIPFSBackend,
+    LocalIPFSBackend,
+    RegistryURIBackend,
+]
+
+
+def resolve_uri_contents(uri: str, fingerprint: bool = None) -> bytes:
+    is_resolvable = get_resolvable_backends_for_uri(uri)
```

Comment:
Just a naming convention thing, but I expected this to be a boolean and then was surprised when I saw it being iterated over.  Maybe something like `backends_for_resolution` or `resolver_backends/translator_backends`.